packages/core/resolve-command/src/index.ts

```diff
@@ -217,7 +297,12 @@ const getAggregateState = async (
           }
 
           log.debug(`loading snapshot`)
-          return JSON.parse(await eventstoreAdapter.loadSnapshot(snapshotKey))
+          const snapshot = await eventstoreAdapter.loadSnapshot(snapshotKey)
+
+          if (typeof snapshot === 'string') {
```

Comment:
Breaking change 2: Eventstore Adapter formerly can return boxed or unboxed string, and now will fail if adapter return boxed string. Correct validation is https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L12135 or simply `snapshot != null && snapshot.constructor === String`